### DISCUSSION TITLE
Power metrics off since Harvest 24.02

### DISCUSSION THREAD

**Discussion by arnaudhirth** [NetApp/harvest#2923] [Q&A]
Hello, since the version 24.02 for ours AFF400 models the power metrics for node still remain with 0 Watts (the shelfs are OK) do you have the same issue?

**Comment by rahulguptajss** [comment]
@arnaudhirth Since Harvest 23.11, you will need to provide an extra REST permission for the relevant user in ONTAP. You can find more details here: https://netapp.github.io/harvest/latest/resources/power-algorithm/

Also, please remember to restart Harvest after granting this permission
